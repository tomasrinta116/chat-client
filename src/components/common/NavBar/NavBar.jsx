import React from "react";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";
import NavMenu from '../NavMenu/NavMenu';
import styles from './NavBar.styles';


const Navbar = props => {
  const {
    classes: { root, grow, menuIcon },
    history, sb, match
  } = props;

  return (
    <div className={root}>
      <AppBar position="static">
        <Toolbar>
          <Typography variant="h6" color="inherit" className={grow}>
            react.chat
          </Typography>
          <div className={menuIcon}>
            <NavMenu history={history} sb={sb} match={match}/>
          </div>
        </Toolbar>
      </AppBar>
    </div>
  );
};

export default withStyles(styles)(Navbar);
