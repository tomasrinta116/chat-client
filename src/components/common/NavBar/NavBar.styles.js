const styles = {
    root: {
      display: "flex"
    },
    menuIcon: {
      marginLeft: "auto"
    }
  };

export default styles; 