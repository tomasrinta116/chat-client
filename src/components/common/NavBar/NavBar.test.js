import React from "react";
import { shallow } from "enzyme";
import NavBar from "./NavBar";

describe("<NavMenu />", () => {
  test("it renders correctly", () => {
    let wrapper = shallow(<NavBar />).dive();
    expect(wrapper).toMatchSnapshot();
  })
})