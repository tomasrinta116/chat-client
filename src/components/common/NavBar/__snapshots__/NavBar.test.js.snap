// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavMenu /> it renders correctly 1`] = `
<div
  className="Navbar-root-1"
>
  <WithStyles(AppBar)
    position="static"
  >
    <WithStyles(Toolbar)>
      <WithStyles(Typography)
        color="inherit"
        variant="h6"
      >
        react.chat
      </WithStyles(Typography)>
      <div
        className="Navbar-menuIcon-2"
      >
        <NavMenu />
      </div>
    </WithStyles(Toolbar)>
  </WithStyles(AppBar)>
</div>
`;
